pkg/minikube/machine/start.go

```diff
@@ -212,23 +213,38 @@ func postStartValidations(h *host.Host, drvName string) {
 	if !driver.IsKIC(drvName) {
 		return
 	}
-	// make sure /var isn't full, otherwise warn
-	output, err := h.RunSSHCommand("df -h /var | awk 'NR==2{print $5}'")
+	r, err := CommandRunner(h)
 	if err != nil {
-		glog.Warningf("error running df -h /var: %v", err)
+		glog.Warningf("error getting command runner: %v", err)
 	}
-	output = strings.Trim(output, "\n")
-	percentageFull, err := strconv.Atoi(output[:len(output)-1])
+	// make sure /var isn't full, otherwise warn
+	percentageFull, err := MemoryCapacity(r, "/var")
 	if err != nil {
 		glog.Warningf("error getting percentage of /var that is free: %v", err)
 	}
 	if percentageFull >= 99 {
-		exit.WithError("", fmt.Errorf("docker daemon out of memory. No space left on device"))
+		exit.WithCodeT(exit.InsufficientStorage, "docker daemon out of memory. No space left on device")
 	}
 
 	if percentageFull > 80 {
-		out.WarningT("The docker daemon is almost out of memory, run 'docker system prune' to free up space")
+		out.ErrT(out.Tip, "The docker daemon is almost out of memory, run 'docker system prune' to free up space")
+	}
+}
+
+// MemoryCapacity returns the capacity of dir in the VM/container
+func MemoryCapacity(cr command.Runner, dir string) (int, error) {
```

Comment:
Probably "StorageCapacity" or something, no ? Memory usually refers to RAM.

Maybe also mention that the returned unit is a percentage (not e.g. bytes)